### PR TITLE
Adds example for process-file

### LINKED ISSUE #75: Add process example to guide
The PR https://github.com/mesg-foundation/docs/pull/68 adds the process guide but without any example.

We could use the example that is already used in the [website](https://mesg.com/mesg-orchestrator/) or this [gist](https://gist.github.com/antho1404/8a02ac0dc79b3e8435fa1495dac0a51e) or create a new abstract example that just shows the different possibilities of the process file https://docs.mesg.com/guide/process/process-file.html.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #75 
Adds example for process-file in `process-file.md`

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@antho1404 please review.

**Comment by agbilotia1998** [comment]
Hi @antho1404 can you please review this PR and let me know if anything else needs to be done?

**Comment by antho1404** [comment]
Hello, sorry for the delay, thanks for the contribution.
That's good to have a complete example at the end but it would be better to have many small examples for each section with their different values.
- Trigger
- Task
- Filter
- Instance resolution

Example:

Trigger:
```yml
key: my-workflow
steps:
  - type: trigger
    instanceHash: XXX
    eventKey: eventX # listen to the event with the key `eventX`
    # or
    taskKey: taskX # listen to the result of the task with the key `taskX`
```

Every example should have the whole structure even if it is truncated.

Example:
```yml
key: my-workflow
steps:
  ...
  - type: filter
    conditions:
      dataX: xxx
  ... 
```

**Comment by agbilotia1998** [comment]
@antho1404 thanks for the review. Sure, I will add the examples for every section.

**Comment by agbilotia1998** [comment]
@antho1404 I have added examples to each of the section, I've added the different possibilities as I understood from the documentation. Please check if I missed something or any changes are required. Thanks :)

File: guide/process/process-file.md [hunk 1]
```diff
@@ -121,6 +133,22 @@ Reference the outputs of a previous step.
...
+steps:
+  ...
+  - type: task
+    instanceHash: "H74Qqq8nT5JZ9GSJmuSWLN5benWZPkUb5pYcvQLsoZX"
+    taskKey: taskY
+    inputs:
+      inputA: "Input1 to the task"
+      # or
+      stepKey: taskX
+      key: taskZ
```
Comment (antho1404) [inline guide/process/process-file.md]: This is a data of `inputA` so should be indented like
```
inputs: 
  inputA:
    stepKey: stepX
    key: taskX
```

This also needs to be updated in the complete example

File: guide/process/process-file.md [hunk 2]
```diff
@@ -156,10 +196,65 @@ All conditions should match to continue to the next step.
   }]
 }" :types="{}" />
 
+#### Example
+```yaml
+key: erc20-notification
+steps:
+  - type: trigger
+    instance:
```
Comment (antho1404) [inline guide/process/process-file.md:205]: One example with the `instanceHash` would be great too like that we have the 3 different versions

**Comment by agbilotia1998** [comment]
@antho1404  I've made the changes, please review.

**Review by antho1404 — APPROVED** [review]
Thanks @agbilotia1998, that's great

**Pull request merged**